.github/workflows/dbld-images.yml

```diff
@@ -0,0 +1,45 @@
+name: Compile dbld-images
+
+on:
+  pull_request:
+  push:
+    paths:
+      - .github/workflows/dbld-images.yml
+      # List is from dbld/rules cache-image-% "WATCHED_FILES"
+      - "dbld/**"
+      - "packaging/rhel/syslog-ng.spec"
+      - "packaging/debian/control"
+  schedule:
+    - cron: '00 03 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_CACHE_NAMESPACE: ghcr.io/${{ github.actor }}
+  
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Build the images
+        run: dbld/rules images
```

Comment:
I think we can give `-j $(nproc)` here, and let `make` run on multiple threads. As we implemented the special image dependencies with `make` rules, we should have a correct build parallelization.
See: https://github.com/syslog-ng/syslog-ng/blob/master/dbld/rules#L210

Reply:
Adding it reduced the build time from 29m 21s to 15m 14s:
 * no `-j`: https://github.com/alltilla/syslog-ng/runs/3629420619?check_suite_focus=true
 * `-j`: https://github.com/alltilla/syslog-ng/runs/3629672133?check_suite_focus=true

Reply:
One thing, we had to think through:

If we build the devshell image, it will trigger the ->tarball->debian-testing images too. Those are also listed in the foreach command in the images target.
So we examined the output from @alltilla earlier run (https://github.com/alltilla/syslog-ng/runs/3629672133?check_suite_focus=true) if GitHub Action (so the makefile) indeed only build those images once.